hera_cal/abscal_funcs.py

```diff
@@ -661,27 +776,46 @@ def UVData2AbsCalDict(filenames, pol_select=None, pop_autos=True, return_meta=Fa
 
     return_meta : boolean, if True: also return antenna and unique frequency and LST arrays
 
+    filetype : string, filetype of data if filenames is a string
```

Comment:
Explicitly mention the valid options. Also say that this is option is ignored if the data is not a string or list of strings.